pkg/kubelet/dockertools/manager.go

```diff
@@ -391,7 +392,14 @@ func makeMountBindings(mounts []kubecontainer.Mount, podHasSELinuxLabel bool) (r
 			} else {
 				bind += ":Z"
 			}
-
+		}
+		// Propagation Mode
+		if len(m.Propagation) != 0 {
+			if !m.ReadOnly && !(m.SELinuxRelabel && podHasSELinuxLabel) {
+				bind += ":" + m.Propagation
+			} else {
+				bind += "," + m.Propagation
```

Comment:
@Kaffa-MY that's my concern. We better ensure the flag is supported before using it.